### PR TITLE
Support snowflake client

### PR DESCRIPTION
**Pull request by pgrivachev**

### CODE REVIEW

File: server/src/snowflake/SnowflakeClient.ts [hunk 1]
```diff
@@ -1,20 +1,88 @@
...
+
+    return new Promise((resolve, reject) => {
+      stream.on('error', e => reject(e));
+      stream.on('end', () => resolve(result));
+    });
+  }
+
+  async getUdf(_projectId: string | undefined, _dataSetId: string, _routineId: string): Promise<Udf | undefined> {
+    await this.connectIfNeeded();
+    throw new Error('Method not implemented.');
```
[the remark this answers is not in the document]
Comment (fivetran-dmitryboykov) [reply server/src/snowflake/SnowflakeClient.ts:57]: It is OK. But it seems that we don't catch this exception.

**Review by fivetran-dmitryboykov — CHANGES_REQUESTED** [review]
Looks good.

File: server/src/snowflake/SnowflakeClient.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (pgrivachev) [reply server/src/snowflake/SnowflakeClient.ts:57]: We don't need to catch it.

[the remark this answers is not in the document]
Comment (pgrivachev) [reply server/src/snowflake/SnowflakeClient.ts:57]: The method is not implemented yet and not used anywhere.

[the remark this answers is not in the document]
Comment (fivetran-dmitryboykov) [reply server/src/snowflake/SnowflakeClient.ts:57]: Ok, just fix it later.

**Review by fivetran-dmitryboykov — APPROVED** [review]

**Pull request merged**